### PR TITLE
Fix cards spacing

### PR DESCRIPTION
**Pull request by StephenAraka**
### What does this PR do?
Fixes issue of project cards not having spacing.
- Changed from `display:flex` to `display:grid` with a `grid-gap`.
**Also**
- it uses the `auto-fill` property to make the grid responsive.

### SCREENSHOTS
**BEFORE**
Screenshot from 2020-06-26 13-24-18

**AFTER - responsive with different scenarios**
Screenshot from 2020-06-26 13-18-35
Screenshot from 2020-06-26 13-18-23
Screenshot from 2020-06-26 13-18-05

### Trello
https://trello.com/c/6wYzhmJg

### CODE REVIEW

File: src/components/UserProjectsPage/UserProjectsPage.css [hunk 1]
```diff
@@ -1,38 +1,30 @@
...
-    flex-direction: row;
-    width: 100%;
-}
-
-.ProjectList .CardImageDiv{
-    background-size: unset!important;
+.ProjectList {
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: repeat(auto-fill, minmax(15rem, 1fr));
```
Comment (mrwagaba) [inline src/components/UserProjectsPage/UserProjectsPage.css:4]: this is not scalable... it has to keep track of the width of the children... let's say u choose to change the project card width to 10rem, u have to change this rule as well

Comment (mrwagaba) [reply src/components/UserProjectsPage/UserProjectsPage.css:4]: here's my suggestion
[
Screenshot 2020-06-29 at 16 43 07
](url)

Comment (mrwagaba) [reply src/components/UserProjectsPage/UserProjectsPage.css:4]: besides, your rule seems complex to comprehend

Comment (mrwagaba) [reply src/components/UserProjectsPage/UserProjectsPage.css:4]: also let the gap be `1rem`

Comment (StephenAraka) [reply src/components/UserProjectsPage/UserProjectsPage.css:4]: Sorry... have just seen this. Let me change the rule and let you know

Comment (StephenAraka) [reply src/components/UserProjectsPage/UserProjectsPage.css]: @mrwagaba I have made the requested changes. Please review.

**Pull request merged**